homeassistant/components/zwave_js/helpers.py

```diff
@@ -4,14 +4,33 @@
 from typing import cast
 
 from zwave_js_server.client import Client as ZwaveClient
+from zwave_js_server.exceptions import NotFoundError
 from zwave_js_server.model.node import Node as ZwaveNode
 
 from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import __version__ as HA_VERSION
 from homeassistant.core import HomeAssistant, callback
 from homeassistant.helpers.device_registry import async_get as async_get_dev_reg
 from homeassistant.helpers.entity_registry import async_get as async_get_ent_reg
 
-from .const import DATA_CLIENT, DOMAIN
+from .const import CONF_DATA_COLLECTION_OPTED_IN, DATA_CLIENT, DOMAIN
+
+
+async def async_enable_statistics(client: ZwaveClient) -> None:
+    """Enable statistics on the driver."""
+    await client.driver.async_enable_statistics("Home Assistant", HA_VERSION)
+
+
+@callback
+def update_data_collection_preference(
+    hass: HomeAssistant, entry: ConfigEntry, preference: bool
+) -> None:
+    """Update data collection preference on config entry."""
+    if not entry:
+        raise NotFoundError("Config entry cannot be found")
```

Comment:
This shouldn't be necessary.

Reply:
could have sworn mypy was not passing because of this but maybe that's when it's not in a decorator